src/rusk/services/echoer.rs

```diff
@@ -0,0 +1,31 @@
+//! Temporary server implementation for rusk
+
+use tonic::{Request, Response, Status};
+
+use basic_proto::echoer_server::Echoer;
+use basic_proto::{EchoRequest, EchoResponse};
+
+pub mod basic_proto {
+    tonic::include_proto!("basic_proto");
+}
+
+#[derive(Debug, Default)]
+pub struct Rusk {}
+
+#[tonic::async_trait]
+impl Echoer for Rusk {
+    async fn echo(
+        &self,
+        request: Request<EchoRequest>, // Accept request of type EchoRequest
+    ) -> Result<Response<EchoResponse>, Status> {
+        // Return an instance of type EchoReply
+        println!("Got a request: {:?}", request);
```

Comment:
We have `tracing` as import, so we should use `info!("Got a request: {:?}", request);` instead

Reply:
Yep, I forgot it!! My bad. 
Changing it!